src/Testing/ErrorFormatterTestCase.php

```diff
@@ -22,9 +22,6 @@ abstract class ErrorFormatterTestCase extends \PHPStan\Testing\TestCase
 
 	private function getOutputStream(): StreamOutput
 	{
-		if (PHP_VERSION_ID >= 80000 && DIRECTORY_SEPARATOR === '\\') {
```

Comment:
Shouldn’t this skip on php 8.x versions which dont‘t yet habe the fix contained?

Reply:
It was fixed during PHP 8 beta development.